src/tribler-core/tribler_core/modules/torrent_checker/torrent_checker.py

```diff
@@ -152,30 +155,42 @@ async def connect_to_tracker(self, session):
             raise e
 
     @db_session
-    def check_random_torrent(self):
+    def torrents_to_check(self):
+        """
+        Two categories of torrents are selected (popular & old). From the pool of selected torrents, a certain
+        number of them are submitted for health check. The torrents that are within the freshness window are
+        excluded from the selection considering the health information is still fresh.
+
+        1. Popular torrents (50%)
+        The indicator for popularity here is considered as the seeder count with direct proportionality
+        assuming more seeders -> more popular. There could be other indicators to be introduced later.
+
+        2. Old torrents (50%)
+        By old torrents, we refer to those checked quite farther in the past, sorted by the last_check value.
+        """
+        four_hours_ago = time.time() - HEALTH_FRESHNESS_SECONDS
```

Comment:
It is 4 hours by default, yes. But it depends on `HEALTH_FRESHNESS_SECONDS` if I understand it correctly. Therefore, maybe we need another name here instead of `four_hours_ago`

Reply:
Yes, I missed that. Thank you for pointing out.